### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by SeekPoint** [mswellhao/chineseNER#3]
rzai@rzai00:~/prj/chineseNER/KG/workDir$ python ../main.py train --train data/traindata --dev data/devdata --test data/testdata --score_dir trainResult
Using gpu device 0: GeForce GTX 1080 (CNMeM is disabled, cuDNN 5105)
choose model type .................. LSTMbaseline
embedding size :  150  embeddings num  :  316001
{'I-PRO': 2, 'B-PRO': 1, 'B-ORG': 9, 'I-COM': 8, 'I-PER': 4, 'B-COM': 7, 'O': 0, 'I-LOC': 6, 'I-ORG': 10, 'B-LOC': 5, 'B-PER': 3}
model training  args : Namespace(CRF=False, action='train', batch_size=64, corpus='data/rawdata', dev='data/devdata', drop_pro=0.35, emb_lrate=0.01, embeddings='vector/baidu_charvectors_150', epoch=10, eva='data/testdata', fix_emb=True, gradbound=-0.1, highreg_weight=0.1, istoken=False, lowreg_weight=0.1, lrate=0.02, model='', model_type='LSTMbaseline', mweight=0.9, nc=3, net_size=[150, 450, 300, 11], opt='adaGrad', output='result', score_dir='trainResult', second_layer='forward', test='data/testdata', train='data/traindata', variance=0.01, wbound=-0.1, wdecay=0.0)
whether fixing word embedding  :  True
model  size :  [150, 450, 300, 11] ;  model  type  :  LSTMbaseline ;  model second layer type  :  forward
adopt sequence based training .............. 
adopt learning method .............  adaGrad
current learning rate ..............0.02
new epoch start.....................0
batch loss  :  143.739120483
batch loss  :  32.5928192139
.............................................................................................
batch loss  :  4.02528572083
batch loss  :  2.54068398476
epoch done ...............   sum loss  :  728.014112115
trainscore ........  epoch  :  3 P:  0.712865819988  R   :   0.399690402476   F  :  0.512200068248

devscore ........  epoch  :  3 P:  0.597014925364  R   :   0.30769230769   F  :  0.406091480781

Segmentation fault (core dumped)
rzai@rzai00:~/prj/chineseNER/KG/workDir$
